frontends/web/src/routes/account/summary/coinbalance.tsx

```diff
@@ -50,6 +56,22 @@ export function CoinBalance ({
   const accountsPerCoin = getAccountsPerCoin();
   const coins = Object.keys(accountsPerCoin) as accountApi.CoinCode[];
 
+  let coinsWithLightning: TCoinMap = {} as TCoinMap;
```

Comment:
If the intention is to re-assign the variable later use `let` else better use `const`.

Reply:
I wasn't sure here, updated.